### ISSUE TITLE
Batch request 

### ISSUE THREAD

**Issue by alberto-bottarini** [theiconic/php-ga-measurement-protocol#30]
Analytics allow to send multiple tracking using a single http request (endpoint http://www.google-analytics.com/batch that accept multiple line, each for a tracking method). 

What do you think about this feature?

**Comment by jorgeborges** [comment]
Looks interesting, it would require quite a bit rewrite tho. PRs accepted otherwise it will be awhile before I get to work on it.

**Comment by alberto-bottarini** [comment]
@jorgeborges 
What do you think about this API?

```php
$analytics
    ->setProtocolVersion('1')
    ->setTrackingId('UA-26293728-11')
    ->setClientId('12345678')
    ->setDocumentPath('/mypage')
    ->setIpOverride("202.126.106.175")
    ->enqueueBatchHit('sendPageView')
    ->setDocumentPath('/mypage2')
    ->enqueueBatchHit('sendPageView')
    ->sendBatchHits();
```

**Comment by alberto-bottarini** [comment]
https://github.com/theiconic/php-ga-measurement-protocol/pull/83

**Comment by jorgeborges** [comment]
Implemented in #90
